ports/esp32s2/common-hal/microcontroller/Pin.c

```diff
@@ -29,18 +29,51 @@
 
 #include "py/mphal.h"
 
+#include "esp-idf/components/driver/include/driver/gpio.h"
+#include "esp-idf/components/soc/include/hal/gpio_hal.h"
+
+STATIC uint32_t never_reset_pins[2];
```

Comment:
It might be nice :-) to just say that these magic number [2] and 32 are just there to create a set or bitmap.

Since it is static ( to this file ) it's probably O.K. to use the idioms/pattern to create a 64-bit map - 
( how do others do this in an abstract yet efficient way )